go/config/config_test.go

```diff
@@ -147,6 +147,10 @@ func TestConfig(t *testing.T) {
 }
 
 func TestUnreadableConfig(t *testing.T) {
+	if os.Getenv("DOCKER") != "" {
+		t.Skip("Skipping testing in Docker environment")
+	}
+
```

Comment:
In docker environment. The following error is occured.
```
--- FAIL: TestUnreadableConfig (0.00s)
	Error Trace:	config_test.go:156
	Error:      	An error is expected but got nil.
	Messages:   	/tmp/home.unreadable/.nomsconfig
```

I can't find out why the test did not pass.
It seems that it can read write-only files in the docker environment.